Buổi 10/homework.js

```diff
@@ -0,0 +1,146 @@
+//Bài 1
+const Multi = number =>{
+    for(i=1;i<11;i++){
+        console.log(`${number} x ${i} = ${number*i}`)
+    }
+}
+
+
+//Bài 2
+const EvenNumber = number =>{
+    if(number<=30){
+        for(i=1;i<=number;i++){
+            if(i%2==0){
+                console.log(i)
+            }
+        }
+    }
+    else{
+        return false
+    }
+}
+
+//Bài 3
+const Sum = number =>{

```

Comment:
```suggestion
// Bài 3
const Sum = number => {
```
Xem comment về cách đặt tên ở bên trên